lib/rubocop/cop/rails/to_s_with_argument.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Identifies passing any argument to `#to_s`.
+      #
```

Comment:
Can you add `@safety` to explain why this cop is unsafe? e.g. https://github.com/rubocop/rubocop-rails/pull/575